vmdb/lib/migration_helper.rb

```diff
@@ -40,33 +40,11 @@ def change_id_columns(table, id_cols, type)
   #
 
   def add_pk(*args)
-    meth = "add_pk_#{connection.adapter_name.downcase}"
-    self.send(meth, *args) if self.respond_to?(meth)
+    # PUNT
```

Comment:
Probably better to just kill the method entirely. That way we can't accidentally end up with something that looks like it'll do a thing, but doesn't. Plus, the resulting NoMethodError will more accurately match the [apparent] current behaviour.